autoload/gina/command/commit.vim

```diff
@@ -302,10 +302,10 @@ function! s:QuitPre() abort
   let args = gina#core#meta#get('args', v:null)
   if args isnot# v:null && get(args.params, 'restore')
     let win_id = win_getid()
-    if bufnr('#') == -1
-      silent keepalt keepjumps 1new
-    else
+    if len(bufname('#')) && bufnr('#') != -1
```

Comment:
1. Please use `!empty(bufname('#'))` instead
2. Please use `isnot#` instead of `!=`